frontend/src/components/utils/DropDown.tsx

```diff
@@ -27,8 +27,22 @@
 import React, { useState } from 'react';
 import { Button, Menu, MenuItem } from '@material-ui/core';
 import { makeStyles } from '@material-ui/styles';
-import { ArrowDropDown, ArrowDropUp } from '@material-ui/icons';
-import SvgIcon from '@material-ui/core/SvgIcon';
+
+const expandArrow = (direction: boolean) => {
+  return (
+    <div style={{ paddingLeft: '10px', transform: direction ? 'rotate(180deg)' : 'none' }}>
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
```

Comment:
I am not sure if this is necessary, but I think creating a local file for the icon/svg in the asset folder and referencing the icon from the asset would be:
1) More flexible for future modification
2) Easier for inspection 
3) Conforming to the general code pattern

Reply:
Thanks for pointing this out! I've updated my code to fix this code format inconsistency in commit 89e93ba.